### ISSUE TITLE
Remove `closable` from tabs

### ISSUE THREAD

**Issue by claviska** [shoelace-style/webawesome#67]
For accessibility, tabs cannot have nested interactives so we need to remove the `closable` attribute and update the docs to show how this can be done with adjacent buttons instead.
